.github/workflows/main.yml

```diff
@@ -27,6 +27,23 @@ env:
   GO_VERSION: 1.16.3
 
 jobs:
+
+  ########################
+  # check PR updates release notes
+  ########################
+  milestone-check:
+    name: check release notes updated
+    runs-on: ubuntu-latest
+    # skip the release note check if the PR is a trivial fix. 
+    if: "!startsWith(github.event.head_commit.message, 'trivial')"
```

Comment:
Do we still want to keep this condition? Perhaps we shouldn't enable both ways ('trivial' prefix and '[skip ci]').